data_entry_helper.php

```diff
@@ -3296,6 +3296,9 @@ public static function species_checklist(array $options) {
     $filterNameTypes = array('all','currentLanguage', 'preferred', 'excludeSynonyms');
     //make a copy of the options so that we can maipulate it
     $overrideOptions = $options;
+    //Set speciesInLabel option on indiciaData  
+    self::$indiciaData["speciesInLabel"] = $options['speciesInLabel'];
```

Comment:
Does this option need documenting in the species_checklist comment?
Also, does it need a default so this doesn't fail if not set when used outside normal dynamic form?

Reply:
@johnvanbreda - Looking at this again to address your comments, I don't think that I have set $indiciaData["speciesInLabel"]  in the right place. It's an option on the [speciemap] control supplied by the dynamic_sample_occurrence form. I'm not sure why I set it from within species_checklist in the data_entry_helper. Should I move it into the dynamic_sample_occurrence form?

Reply:
Yes, that makes more sense.

Reply:
@johnvanbreda - I've moved the setting of $indiciaData["speciesInLabel"] into dynamic_sample_occurrence. If you are happy with everything, I will update the user docs for the [speciesMap] control.